sunpy/net/dataretriever/sources/tests/test_lyra_ud.py

```diff
@@ -19,21 +22,22 @@ def LCClient():
 
 @pytest.mark.remote_data
 @pytest.mark.parametrize("timerange,url_start,url_end", [
-    (TimeRange('2012/1/7', '2012/1/7'),
+    (Time('2012/1/7', '2012/1/7'),
```

Comment:
Does this return a timerange?

Reply:
It was `from sunpy.net.attrs import Time` So I just used it.

Reply:
Fair enough.